libpicotcp/CMakeLists.txt

```diff
@@ -67,3 +72,21 @@ if(LibPicotcp)
     set_property(TARGET picotcp PROPERTY INTERFACE_LINK_LIBRARIES picotcp_sources)
     target_include_directories(picotcp INTERFACE "${SOURCE_DIR}/build/include")
 endif()
+if(LibPicotcpBsd)
```

Comment:
Hey, minor request: could you nest this if block (`LibPicotcpBsd`) within the previous block (`LibPicotcp`)?

Reply:
Ah, please ignore this comment - I see what happened here, this isn't your change

Reply:
I'll just pull in the branch on my account and merge it -- please consider this PR to be closed; sorry for the turbulence but I'm mostly new to dealing with PRs on Github (I've used Bitbucket so far) so I'm missing details in the interface.

Reply:
https://github.com/seL4/util_libs/pull/18 <-- If you're fine with the final state here, @fabriziobertocci , I'm fine with merging~